workflow/workflow-demos/processes/Trigger/ParkingLotReservation.p.json

```diff
@@ -89,6 +89,7 @@
           "  <%=ivy.cms.co(\"/Emails/confirmParkingLot\")%>",
           "</html>"
         ],
+        "contentType" : "text/html",

```

Comment:
here's a rare case of a real conversion: mail-content is html, therefore we set it's mimeType accordingly

Reply:
👍